OmiseSDK/Capability.swift

```diff
@@ -61,6 +61,7 @@ extension Capability {
             case card(Set<CardBrand>)
             case installment(PaymentInformation.Installment.Brand, availableNumberOfTerms: IndexSet)
             case internetBanking(PaymentInformation.InternetBanking)
+            case mobileBanking
```

Comment:
I assume there are more `mobileBanking` payment methods coming later? So we should probably create a new `PaymentInformation` object and the `case` should look like this:
```
case mobileBanking(PaymentInformation.MobileBanking)
```

Reply:
I noticed this was missing from the PR; will update now.